allensdk/brain_observatory/vbn_2022/metadata_writer/lims_queries.py

```diff
@@ -623,6 +668,14 @@ def _behavior_session_table_from_ecephys_session_id_list(
                 on='foraging_id',
                 how='left')
 
+    # The date_of_acquisition and session_type stored in the LIMS
+    # behavior_sessions table is untrustworthy. We will set those
+    # columns to None here so that they all get patched from the
+    # pickles files.
```

Comment:
making me hungry

Reply:
Pickles have long been one of my favorite foods. They are currently one of my least favorite files ;)